src/components/Contacts/ContactDetails/ContactDetailsHeader/ContactDetailsHeader.graphql

```diff
@@ -15,6 +15,38 @@ fragment ContactDetailsHeader on Contact {
   ...ContactHeaderNewsletter
 }
 
+fragment ContactDetails on Contact {
```

Comment:
Also, can you delete this fragment entirely? It appears that the other fragments are loading all of the data that the components need.